sections/problem-analysis/summary.tex

```diff
@@ -0,0 +1,6 @@
+\section{Summary of the problem analysis}
+In the preceding chapter we examined some existing solutions related to the problem of making accessible teaching services.
+This revealed some differences in the ways tutors set their prices, and a correlation between the possibility to upload materials and receiving personal tutoring.
+We defined a set of personas, which are fictive profiles of possible users, to help when constructing the design.
+We determined some different types of scalability, and found some important principles to follow when making a system with scalability in mind, such as writing asynchronous code and building the API first.
+Finally we defined our process, basing it around short, weekly sprints.
```

Comment:
Kunne godt tænke mig at der stod mere med hvad det faktisk indeholder, sådan at man nærmest bare kan læse summary og have den viden der er i rapporten.

fx. We determined that scalability can be split into the x categories a, b, c and d........ and found some important principles such as......